weather-app-master/README.md

```diff
@@ -0,0 +1,92 @@
+<!-- Please update value in the {}  -->
```

Comment:
You should also rename this folder from `weather-app-master` to something more appropriate